spec/core/data_structures.md

```diff
@@ -469,6 +480,13 @@ func SumTruncated(bz []byte) []byte {
 | commit | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Parameter that configures how long Tendermint will wait after receiving a quorum of precommits before beginning consensus for the next height.| 5 |
 | bypass_commit_timeout | bool | Parameter that, if enabled, configures the node to proceed immediately to the next height once the node has received all precommits for a block, forgoing the commit timeout. |  6  |
 
+## ABCIParams
+
+| Name                            | Type  | Description                                                                                      | Field Number |
+|---------------------------------|-------|--------------------------------------------------------------------------------------------------|---|
+| vote_extensions_enable_height   | int64 |                                                                                                  | 1 |
```

Comment:
[Here](https://github.com/tendermint/tendermint/blob/331860c2a8c406fa89575165322b963925fb38c5/spec/abci%2B%2B/abci%2B%2B_app_requirements.md#abciparamsvoteextensionsenableheight) is a description of the flag.